pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/LoadManager.java

```diff
@@ -134,9 +134,7 @@ static LoadManager create(final PulsarService pulsar) {
             // Assume there is a constructor with one argument of PulsarService.
             final Object loadManagerInstance = loadManagerClass.newInstance();
             if (loadManagerInstance instanceof LoadManager) {
-                final LoadManager casted = (LoadManager) loadManagerInstance;
-                casted.initialize(pulsar);
-                return casted;
+                return (LoadManager) loadManagerInstance;
```

Comment:
Why this is changed? And how is it related to delayed delivery?

Reply:
Yup. this was a mistake. It was supposed to add the loader for the tracker without touching this.